packages/ios_platform_images/README.md

```diff
@@ -16,20 +16,13 @@ When loading images from Image.xcassets the device specific variant is chosen
 
 ### iOS->Flutter Example
 
-``` dart
-// Import package
-import 'package:ios_platform_images/ios_platform_images.dart';
-
-Widget build(BuildContext context) {
-  return MaterialApp(
-    home: Scaffold(
-      body: Center(
-        child: Image(image: IosPlatformImages.load("flutter")),
-      ),
-      //..
-    ),
-  );
-}
+<?code-excerpt "example/lib/main.dart (Usage)"?>
+```dart
+// "flutter" is a resource in Assets.xcassets.
```

Comment:
Maybe we should rename the asset rather than explain the name with a comment.

Reply:
What would we name it that wouldn't require an explanation of where the asset lives?

Reply:
"asset"?

Reply:
or something more descriptive

Reply:
In the context of Flutter code, an asset is usually a *Flutter* asset, which is a similar concept but in Flutter-land (specifically, in pubspec.yaml) rather than Xcode-project-land; that's why there's a comment explaining it precisely. I guess we could name it `'assetFromAssets.xcassets'`, but I think that's harder to read than the comment.

Reply:
Whatever you think is best, that's why it's optional :)

Reply:
I can't come up with anything that I find clearer than the comment, so I'll land as-is.